components/deploy/StaticPreview.tsx

```
@@ -0,0 +1,255 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useDeployContext } from '@/providers/DeployProvider';
+
+function parseImports(code: string) {
+  // Matches: import X from 'pkg'; import {A, B} from 'pkg'; import * as X from 'pkg';
+  // Also matches local, CSS, and type-only imports
+  // Updated regex to match scoped packages (e.g., @radix-ui/react-slot)
+  const importRegex = /import\s+((\w+)|\*\s+as\s+(\w+)|\{([^}]+)\})?\s*from\s*['"]([^'"]+)['"];?|import\s+['"]([^'"]+\.css)['"];?|import\s+type\s+\{[^}]+\}\s+from\s+['"]([^'"]+)['"];?/g;
+  let match;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)